src/app/tests/suites/certification/Test_TC_SC_5_2.yaml

```diff
@@ -76,11 +76,11 @@ tests:
                     {
                         GroupKeySetID: 0x01a3,
                         GroupKeySecurityPolicy: 0,
-                        EpochKey0: "0xd0d1d2d3d4d5d6d7d8d9dadbdcdddedf",
+                        EpochKey0: "\xd0\xd1\xd2\xd3\xd4\xd5\xd6\xd7\xd8\xd9\xda\xdb\xdc\xdd\xde\xdf",
```

Comment:
Could we do `"hex:d0d1d2d3d4d5d6d7d8d9dadbdcdddedf"` for slightly better readability?